TA_dataset/lib/dataset_query_api_client/api/default/post_queries.py

```diff
@@ -21,6 +21,7 @@ def _get_kwargs(
     *,
     client: Client,
     json_body: PostQueriesLaunchQueryRequestBody,
+    tenant_details=None,
```

Comment:
imho no tenant_details is needed as PostQueriesLaunchQueryRequestBody already contains field additional_properties which should be sufficient